homeassistant/components/config/config_entries.py

```diff
@@ -287,6 +268,30 @@ async def system_options_update(hass, connection, msg):
     connection.send_result(msg["id"], entry.system_options.as_dict())
 
 
+@websocket_api.require_admin
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {"type": "config_entries/update", "entry_id": str, vol.Optional("title"): str}
+)
+async def config_entry_update(hass, connection, msg):
+    """Update config entry system options."""
+    changes = dict(msg)
+    changes.pop("id")
+    changes.pop("type")
+    entry_id = changes.pop("entry_id")
+
+    entry = hass.config_entries.async_get_entry(entry_id)
+
+    if entry is None:
```

Comment:
Maybe test this case?

Reply:
https://github.com/home-assistant/core/pull/34191